zqd/handlers_zeek_test.go

```diff
@@ -1,11 +1,11 @@
-// +build zeek
-
```

Comment:
With the changes here, we're no longer testing zeek, right? That seems like a big step backward to me.

Reply:
We are testing the functionality surrounding zeek by passing a `testProcess` into `core` that simulates what a zeek process would do. This is better for unit tests, which these endpoint tests are. I agree that we will need some kind of integration/system test w/ live zeek, but even still its hard to replicate some of the error cases we can do here with a mocked zeek process.